src/main/scala/com.snowplowanalytics.iglu/ctl/commands/VerifyParquet.scala

```diff
@@ -0,0 +1,79 @@
+package com.snowplowanalytics.iglu.ctl.commands
+
+import cats.Show
+import cats.data.{EitherT, NonEmptyList}
+import cats.effect.IO
+import cats.implicits._
+import com.snowplowanalytics.iglu.core.{SchemaKey, SelfDescribingSchema}
+import com.snowplowanalytics.iglu.ctl._
+import com.snowplowanalytics.iglu.schemaddl.jsonschema.Schema
+import com.snowplowanalytics.iglu.schemaddl.jsonschema.circe.implicits.toSchema
+import com.snowplowanalytics.iglu.schemaddl.parquet.{Field, Migrations}
+import io.circe._
+
+import java.nio.file.Path
+
+object VerifyParquet {
+
+  def process(command: Command.VerifyParquet): Result = {
+    readSchemas(command.input)
+      .map(handleInputSchemas)
+      .map(prepareOutputMessage)
+  }
+
+  private def readSchemas(input: Path): FailingNel[List[SelfDescribingSchema[Schema]]] = {
+    EitherT(File.readSchemas(input).map(Common.leftBiasedIor))
+      .flatMap { files => 
+        files.toList
+          .traverse(schemaFile => toIgluSchema(schemaFile.content))
+      }
+  }
+
+  private def handleInputSchemas(schemas: List[SelfDescribingSchema[Schema]]) = {
+    groupSchemasByModel(schemas)
+      .flatMap(detectBreakingChanges)
+  }
+
+  private def groupSchemasByModel(schemas: List[SelfDescribingSchema[Schema]]): List[List[SelfDescribingSchema[Schema]]] = {
+    schemas
+      .groupBy { schema =>
+        (schema.self.schemaKey.vendor, schema.self.schemaKey.name, schema.self.schemaKey.version.model)
+      }.values.toList
+  }
+
+  private def detectBreakingChanges(schemas: List[SelfDescribingSchema[Schema]]): List[BreakingChange] = {
+    schemas.zip(schemas.tail)
+      .filter(isBreaking)
+      .map { breakingPair =>
+        BreakingChange(breakingPair._1.self.schemaKey, breakingPair._2.self.schemaKey)
+      }
+  }
+
+  private def isBreaking(pair: (SelfDescribingSchema[Schema], SelfDescribingSchema[Schema])): Boolean = {
```

Comment:
Schema family must be merged, taking only pairs would miss breaking changes that were accumulated from older schemas.

Check how it is done in the [transfomer](https://github.com/snowplow/snowplow-rdb-loader/blob/master/modules/common/src/main/scala/com/snowplowanalytics/snowplow/rdbloader/common/transformation/parquet/NonAtomicFieldsProvider.scala#L67).